### DISCUSSION TITLE
"Popular repositories" is probably irrelevant

### DISCUSSION THREAD

**Discussion by evgeniikozhanov** [community/community#49436] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The "Popular repositories" is probably irrelevant. I mean a block in the overview tab on the page like this https://github.com/mycompany
How do you define them? by count of forks?

**Comment by radar-blip** [comment]
By number of stars.

**Reply by evgeniikozhanov** [reply]
in my case all showing repos are forked, have no stars, may the number of stars of parent repos have weight?

**Reply by radar-blip** [reply]
It's a possibility, but I'm not sure.
